<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaMat</class>
 <widget class="QDialog" name="NuevaMat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTimeEdit" name="timeFin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ldesde">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde</string>
   </property>
  </widget>
  <widget class="QLabel" name="lhasta">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeInicio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNombre">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string comment="Ejemplo" extracomment="Ejemplo"/>
   </property>
   <property name="inputMask">
    <string comment="Ejemplo" extracomment="Ejemplo"/>
   </property>
   <property name="text">
    <string comment="Ejemplo" extracomment="Ejemplo"/>
   </property>
   <property name="placeholderText">
    <string comment="Ejemplo" extracomment="Ejemplo"/>
   </property>
  </widget>
  <widget class="QLabel" name="lnombre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;B&gt;&lt;/B&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lAula">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;B&gt;Aula:&lt;/B&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxAulas">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
